src/main/java/de/tum/in/www1/artemis/service/scheduled/cache/CacheHandler.java

```diff
@@ -20,6 +20,8 @@ public abstract class CacheHandler<K, C extends Cache> {
     protected final IMap<K, C> cache;
 
     protected CacheHandler(HazelcastInstance hazelcastInstance, String name) {
+        // NOTE: it's ok to directly access hazelcastInstance in the constructor, because this is no Bean/Service and will be created after the services are initialized in
```

Comment:
services are initialized in ... ?

Reply:
i think the "in" is unnecessary, will remove it